webapp/helpers.py

```diff
@@ -25,17 +25,30 @@ def __init__(self, url_map, *items):
 
 def get_licenses():
     try:
-        with open("webapp/licenses.json") as f:
-            licenses_file = json.load(f)
-
-        licenses = licenses_file["licenses"]
+        with open("webapp/store-licenses.json") as f:
+            licenses = json.load(f)
 
         def _build_custom_license(license_id, license_name):
             return {"licenseId": license_id, "name": license_name}
 
         CUSTOM_LICENSES = [
```

Comment:
@Lukewh I am just trying to understand this, why these licenses defined here are not in the json file so we don't need to define them here?

Reply:
The json file is a subset of https://spdx.org/licenses/, the custom licenses are ones we're adding specifically for snapcraft so this is just to keep them separate. The way this worked and how it will again one day, is that the licenses.json (spdx v3) is downloaded automatically on app start which would overwrite our custom licenses - so this is both left over from before and also prep for the future.